x-pack/test/functional/apps/ml/anomaly_detection_result_views/anomaly_explorer.ts

```diff
@@ -85,495 +110,531 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
+            });
           });
         });
+      }
+    });
+    describe('with no influencers', function () {
+      for (const testData of testDataListWithNoInfluencers) {
+        before(async () => {
+          await ml.api.createAndRunAnomalyDetectionLookbackJob(
+            testData.jobConfig,
+            testData.datafeedConfig
+          );
+        });
 
-        describe('Use anomaly table action to view in Discover', function () {
-          beforeEach(async () => {
-            await ml.navigation.navigateToAnomalyExplorer(
-              testData.jobConfig.job_id,
-              {
-                from: '2016-02-07T00%3A00%3A00.000Z',
-                to: '2016-02-11T23%3A59%3A54.000Z',
-              },
-              () => elasticChart.setNewChartUiDebugFlag(true)
-            );
+        after(async () => {
+          await elasticChart.setNewChartUiDebugFlag(false);
+          await ml.api.cleanMlIndices();
+        });
 
-            await ml.commonUI.waitForMlLoadingIndicatorToDisappear();
-            await ml.commonUI.waitForDatePickerIndicatorLoaded();
-            await ml.swimLane.waitForSwimLanesToLoad();
-          });
+        it('should not display the influencers panel', async () => {
+          await ml.testExecution.logTestStep('navigate to job list');
+          await ml.navigation.navigateToMl();
+          // Set debug state has to happen at this point
+          // because page refresh happens after navigation to the ML app.
+          await elasticChart.setNewChartUiDebugFlag(true);
+          await ml.navigation.navigateToJobManagement();
 
-          it('should render the anomaly table', async () => {
-            await ml.testExecution.logTestStep('displays the anomalies table');
-            await ml.anomaliesTable.assertTableExists();
+          await ml.testExecution.logTestStep('open job in anomaly explorer');
+          await ml.jobTable.filterWithSearchString(testData.jobConfig.job_id, 1);
 
-            await ml.testExecution.logTestStep('anomalies table is not empty');
-            await ml.anomaliesTable.assertTableNotEmpty();
-          });
+          await ml.jobTable.clickOpenJobInAnomalyExplorerButton(testData.jobConfig.job_id);
+          await ml.commonUI.waitForMlLoadingIndicatorToDisappear();
+          await ml.jobSelection.assertJobSelection([testData.jobConfig.job_id]);
 
-          it('should click the Discover action in the anomaly table', async () => {
-            await ml.anomaliesTable.assertAnomalyActionsMenuButtonExists(0);
-            await ml.anomaliesTable.scrollRowIntoView(0);
-            await ml.anomaliesTable.assertAnomalyActionsMenuButtonEnabled(0, true);
-            await ml.anomaliesTable.assertAnomalyActionDiscoverButtonExists(0);
-            await ml.anomaliesTable.ensureAnomalyActionDiscoverButtonClicked(0);
-          });
+          await ml.testExecution.logTestStep('displays the influencers list');
```

Comment:
Nit: Should this say `does not display the influencers list`?

Reply:
good catch!
updated